<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QC Tool&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Docx2TEI</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">qproperty-alignment: AlignCenter;</string>
          </property>
          <property name="text">
           <string>prepend text</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="open_word_dir">
          <property name="text">
           <string>open docx directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="run_transformation">
          <property name="styleSheet">
           <string notr="true">margin-bottom: 50px;
margin-top: 20px;
padding: 5px;</string>
          </property>
          <property name="text">
           <string>run transformation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPlainTextEdit" name="prepend_text_edit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="open_output_dir">
          <property name="text">
           <string>set output</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="6">
         <widget class="QTextEdit" name="textEdit">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>open_word_dir</zorder>
       <zorder>open_output_dir</zorder>
       <zorder>label</zorder>
       <zorder>textEdit</zorder>
       <zorder>prepend_text_edit</zorder>
       <zorder>run_transformation</zorder>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>QC Tool</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="directory">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="results">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">margin-top: 25px;
margin-bottom: 5px;</string>
          </property>
          <property name="text">
           <string>Results:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="dir_open">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>open directory</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="QListWidget" name="list_widget">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>17</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QPushButton" name="export_2">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>export report</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="check_box">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
           </font>
          </property>
          <property name="text">
           <string>Fix Errors</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="run_report">
          <property name="font">
           <font>
            <family>Helvetica Neue</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>run report</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
